earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +51,12 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+fusiontestdir = os.path.join(testdir, 'fusion')
+shutil.rmtree(fusiontestdir, ignore_errors=True)
+shutil.copytree('../../fusion', fusiontestdir)
```

Comment:
This seems will copy src/fusion to ${bin}/tests/fusion? Why do we need this?
I couldn't find an issue for this fix,
Can you please add an issue, so it will be easier to figure out what this fix is for?
Thanks.